Generated code reassigns to const

The generated code contains the following:

```js
class ComponentAsyncState {
  // ...
  wakeNextTaskForAwaitable(awaitable) {
    // ...

    // If there is an active promise, then wait for it
    while (this.#lock.promise) {
      const finishedTicket = await this.#lock.promise;
      finishedTicket = await this.#lock.promise;
      if (finishedTicket === ticket - 1n) { break; }
    }
```

This fails to build in a bundler because one cannot reassing to the `const finishedTicket`.

Specifically the error message from `esbuild`:
```
✘ [ERROR] Cannot assign to "finishedTicket" because it is a constant

    src/generated/delta_wit_serialize.js:593:4:
      593 │     finishedTicket = await this.#lock.promise;
          ╵     ~~~~~~~~~~~~~~

  The symbol "finishedTicket" was declared a constant here:

    src/generated/delta_wit_serialize.js:592:10:
      592 │     const finishedTicket = await this.#lock.promise;
```

The issue has been introduced by https://github.com/bytecodealliance/jco/pull/817 which landed in `1.13.1`.
The quick fix is to downgrade to `1.13.0`.

--- a/crates/js-component-bindgen/src/intrinsics/component.rs
+++ b/crates/js-component-bindgen/src/intrinsics/component.rs
@@ -160,8 +160,8 @@ impl ComponentIntrinsic {
                             }});
 
                             // If there is an active promise, then wait for it
+                            let finishedTicket;
                             while (this.#lock.promise) {{
-                                const finishedTicket = await this.#lock.promise;
                                 finishedTicket = await this.#lock.promise;
                                 if (finishedTicket === ticket - 1n) {{ break; }}
                             }}

